crates/astria-sequencer/src/bridge/state_ext.rs

```diff
@@ -615,6 +618,81 @@ mod test {
         );
     }
 
+    #[tokio::test]
+    async fn get_deposit_rollup_ids() {
```

Comment:
I think the extra check in the middle of this test is redundant. IMO just put 3 deposits, 1 of them a duplicate. In the end, ensure that a) final result has `len == 2`, and b) that rollup IDs are in the set.